src/desktop/apps/artsy-v2/apps/artwork/artworkClient.tsx

```diff
@@ -10,15 +9,23 @@ import {
 } from "typings/mediator"
 
 export const artworkClient = () => {
+  require("jquery.transition")
+  require("typeahead.js/dist/typeahead.bundle.min.js")
+  const $ = require("jquery")
+  const imagesLoaded = require("imagesloaded")
+  const jqueryFillwidthLite = require("jquery-fillwidth-lite")
+
+  imagesLoaded.makeJQueryPlugin($)
+  jqueryFillwidthLite($, window._, imagesLoaded)
```

Comment:
Is jquery all we use?

Reply:
nah, without these plugins it blows up for some reason

Reply:
Yeah, I was asking if these are the only extra dependencies that are needed. Is backbone also required?

Reply:
The inquiry flow is all built around backbone unfortunately